frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/PipelineSecretSection.tsx

```diff
@@ -98,7 +98,7 @@ const PipelineSecretSection: React.FC = () => {
               className="odc-pipeline-secret-section__secret-action"
               icon={<PlusCircleIcon />}
             >
-              {t('pipelines-plugin~Add Secret')}
+              {t('pipelines-plugin~Add secret')}
```

Comment:
Secret is a k8s resource. Capital is correct I believe.

cc @bgliwa01

Reply:
Just fyi:

![image](https://user-images.githubusercontent.com/139310/122005127-8b9c8f00-cdb5-11eb-9f8f-88d72032862c.png)

But I also know that Secret is here "exact a Secret resource" and the values in the dropdown are talking about the concept "secret" ... Anyway a little bit confusing.

Reply:
When secret is referred to as a resource it should be capital

Reply:
Done